Translated rag_zephyr_langchain.ipynb to Korean

jun048098 opened the pull request:
# What does this PR do?

<!--
Thank you for submitting a PR to the Open-source AI cookbook!

Someone will review your PR shortly. They may suggest changes to further improve your contribution. 
If no one reviewed your PR after a week has passed, don't hesitate to post a new comment @-mentioning the same 
persons---sometimes notifications get lost.

-->
This is PR presents the Korean translation of [Simple RAG for GitHub issues using Hugging Face Zephyr and LangChain](https://huggingface.co/learn/cookbook/rag_zephyr_langchain) cookbook
<!-- Remove if not applicable -->

Fixes # (issue)

## Who can review?

@4N3MONE, @jeongiin @boyunJang, @ahnjj, @SeungAhSon, @jun048098, @win2dvp21 @fabxoe 


May you please review this PR? @merveenoyan and @stevhliu.

4N3MONE commented on notebooks/ko/rag_eeve_langchain.ipynb:
```diff
@@ -0,0 +1 @@
+{"cells":[{"cell_type":"markdown","metadata":{"id":"Kih21u1tyr-I"},"source":["# GitHub 이슈를 위한 EEVE와 LangChain을 사용한 간단한 RAG\n","\n","본문: [Maria Khalusova](https://github.com/MKhalusova)의 [Simple RAG for GitHub issues using Hugging Face Zephyr and LangChain](https://huggingface.co/learn/cookbook/rag_zephyr_langchain)을 번역하고, 한국어 모델로 변경했습니다.\n","\n","번역: [신혁준](https://github.com/jun048098)\n","\n","이 ...
```
<p>제안 : 디스틸 버전 -&gt; 증류된 모델, 증류된 버전</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/214/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

jun048098 replied:
<p>수정했습니다</p>

4N3MONE commented on notebooks/ko/rag_eeve_langchain.ipynb:
```diff
@@ -0,0 +1 @@
+{"cells":[{"cell_type":"markdown","metadata":{"id":"Kih21u1tyr-I"},"source":["# GitHub 이슈를 위한 EEVE와 LangChain을 사용한 간단한 RAG\n","\n","본문: [Maria Khalusova](https://github.com/MKhalusova)의 [Simple RAG for GitHub issues using Hugging Face Zephyr and LangChain](https://huggingface.co/learn/cookbook/rag_zephyr_langchain)을 번역하고, 한국어 모델로 변경했습니다.\n","\n","번역: [신혁준](https://github.com/jun048098)\n","\n","이 ...
```
<p>한국어 리더보드(<a href="https://huggingface.co/spaces/upstage/open-ko-llm-leaderboard" target="_blank">https://huggingface.co/spaces/upstage/open-ko-llm-leaderboard</a>, <a href="https://lk.instruct.kr" target="_blank">https://lk.instruct.kr/</a>) 로 링크를 수정하는건 어떨까요?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/214/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

jun048098 replied:
<p>수정했습니다.</p>

4N3MONE commented on notebooks/ko/rag_eeve_langchain.ipynb:
```diff
@@ -0,0 +1 @@
+{"cells":[{"cell_type":"markdown","metadata":{"id":"Kih21u1tyr-I"},"source":["# GitHub 이슈를 위한 EEVE와 LangChain을 사용한 간단한 RAG\n","\n","본문: [Maria Khalusova](https://github.com/MKhalusova)의 [Simple RAG for GitHub issues using Hugging Face Zephyr and LangChain](https://huggingface.co/learn/cookbook/rag_zephyr_langchain)을 번역하고, 한국어 모델로 변경했습니다.\n","\n","번역: [신혁준](https://github.com/jun048098)\n","\n","이 ...
```
<p>한국어 질문이면 더욱 좋을 것 같습니다!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/214/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

jun048098 replied:
<p>수집한 PEFT 라이브러리 이슈가 영어로 작성되어서 리트리버에서 한글로 검색이 어렵습니다. ㅠㅠ</p><p>마크다운에 영어 질문 이유를 추가 설명했습니다.</p>

4N3MONE commented on notebooks/ko/rag_eeve_langchain.ipynb:
```diff
@@ -0,0 +1 @@
+{"cells":[{"cell_type":"markdown","metadata":{"id":"Kih21u1tyr-I"},"source":["# GitHub 이슈를 위한 EEVE와 LangChain을 사용한 간단한 RAG\n","\n","본문: [Maria Khalusova](https://github.com/MKhalusova)의 [Simple RAG for GitHub issues using Hugging Face Zephyr and LangChain](https://huggingface.co/learn/cookbook/rag_zephyr_langchain)을 번역하고, 한국어 모델로 변경했습니다.\n","\n","번역: [신혁준](https://github.com/jun048098)\n","\n","이 ...
```
<p>제안 : 반복에서는 -&gt; 반복해서 RAG를 할때는, 반복 작업에서는</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/214/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

jun048098 replied:
<p>수정했습니다.</p>

merveenoyan commented:
It's best to keep the filename `rag_zephyr_langchain.ipynb` under ko 😊 @jun048098  고맙습니다
I checked using translation, LGTM if you think as well, after the conflicts are resolved we can merge

jun048098 commented:
> It's best to keep the filename `rag_zephyr_langchain.ipynb` under ko 😊 @jun048098 고맙습니다 I checked using translation, LGTM if you think as well, after the conflicts are resolved we can merge

The original text used Hugging Face Zephyr. In translation, eeve was used to use the Korean model. For this reason, I changed the ipynb file name.

However, I changed it back to the original rag_zephyr_langchain.ipynb. 
Also, the conflict was resolved.

@merveenoyan thank you!

merveenoyan commented:
@jun048098 it's nice you adapted it, I feel like it's best to keep the equivalent of the exact same one so people can switch between languages. If you can get one of the fellow Koreans to approve we can merge (also there are some merge conflicts due to recently merged Korean recipes)

fabxoe reviewed (approved):
LGTM!

jun048098 commented:
@merveenoyan @stevhliu Hello! I received an apporve from a Korean reviewer. Is merging possible? thank you

merveenoyan commented:
@jun048098 feel free to join this org: https://huggingface.co/huggingcooks

HuggingFaceDocBuilderDev commented:
The docs for this PR live [here](https://moon-ci-docs.huggingface.co/docs/cookbook/pr_214). All of your documentation changes will be reflected on that endpoint. The docs are available until 30 days after the last update.
